home_task_queries.sql

```diff
@@ -0,0 +1,133 @@
+/*
+ Завдання на SQL до лекції 02.
+ */
+
+
+/*
+1.
+Вивести кількість фільмів в кожній категорії.
+Результат відсортувати за спаданням.
+*/
+with filmCountByFilmCategory(category_id, film_count) as
+         (select category_id,
+                 count(*) as film_count
+          from film_category
+          group by category_id)
+
+select c.name, s.film_count
+from filmCountByFilmCategory as s
+         left join category as c on s.category_id = c.category_id
+order by s.film_count desc
```

Comment:
This query looks slightly over-complicated. 
It could be simple, for instance:
```
select
    category.name AS category,
    count(film_category.film_id)  AS films_amount
from category
         left join film_category on category.category_id = film_category.category_id
group by category.name
order by films_amount desc;
```